src/main/java/lotto/model/result/LottoStatistics.java

```diff
@@ -0,0 +1,30 @@
+package lotto.model.result;
+
+import java.util.Map;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+import lotto.model.money.Money;
+
+public class LottoStatistics {
+
+    private final Map<LottoRank, Long> lottoRankCountMap;
+
+    public LottoStatistics(LottoRanks ranks) {
+        this.lottoRankCountMap = ranks.getLottoRanks().stream()
+                .collect(Collectors.groupingBy(Function.identity(), Collectors.counting()));
+    }
+
+    public long count(LottoRank rank) {
+        if (lottoRankCountMap.get(rank) != null) {
+            return lottoRankCountMap.get(rank);
+        }
+        return 0;
```

Comment:
```suggestion
        if (!lottoRankCountMap.contains(rank)) {
            throw...
        }
        return lottoRankCountMap.get(rank); 
```
개인적으로 `return 0;` 같은 방어적코드 보단 throw를 내서 빠른 피드백을 받는형태를 선호해요.

엘레강트 오브젝트에 '절대 NULL을 반환하지 마세요' 부분을 읽어보시면 좋을  것 같아요!

Reply:
제가 코드를 가독성 좋지 않게 작성했던 것 같아요! 
여기서 0은 방어적 코드라기보다는 `lottoRankCounter`에서 해당 등수의 당첨된 것이 없으면 0개 당첨됐다고 반환해주는 값입니다!